doc/man7/EVP_RAND-HASH-DRBG.pod

```diff
@@ -88,6 +105,11 @@ NIST SP 800-90A and SP 800-90B
 L<EVP_RAND(3)>,
 L<EVP_RAND(3)/PARAMETERS>
 
+=head1 HISTORY
+
+OpenSSL 3.1.1 restricted the permitted digests when using the FIPS provider as per 
```

Comment:
This PR is also marked for 3.0, so don't you also need to mention the 3.0 patch that introduces this?

Reply:
Only the first (tests) commit is for 3.0.

Reply:
As per my earlier [comment](https://github.com/openssl/openssl/pull/20521#issuecomment-1471249422), for 3.0, it is **only** the test data being updated so we can test 3.0 with a later FIPS provider.  So no, we don't need to mention anything for 3.0.

This is a cost we have to pay as part of the cross version FIPS provider testing.  We need to update the policy to allow this without having to vote an exception every time.